README.md

```diff
@@ -54,4 +87,35 @@ All debug statements must be removed for the optimized build.
 
 Environment variables are stored `.env.local` for local development, `.env.devel` for the development server build, and `.env.production` for the production build.
 
-There may be occassional caching problems when using these environments. Deleting `.cache/` and `dist/` will usually help.
\ No newline at end of file
+There may be occassional caching problems when using these environments. Deleting `.cache/` and `dist/` will usually help.
+
+## Server
+
+### Virtual Environment
+
+You'll need to create a virtual environment to keep the dependencies for this project contained. With `virtualenv` installed, use the following commands.
+
+```
+virtulenv -p python3 venv
+source venv/bin/activate
+pip install -r requirements.txt
+```
+
+### Develop
+Be sure to have your virtualized environment activated. From the project's root directory run:
+
+```
+python manage.py runserver
+```
+
+## Build
+
+Build [![CircleCI](https://circleci.com/gh/ereadingtool/ereadingtool.svg?style=svg)](https://circleci.com/gh/ereadingtool/ereadingtool)
```

Comment:
We might want to keep the badge up top. Nice to see the build status when you first see the README.

Reply:
Oh that's what that is! To the top it goes.